### PR TITLE
Add Chrome 98 to 100

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
The data changes are:
- _static class methods use [[Define]]_ is fixed in Chrome 98 (https://github.com/babel/babel/pull/14295#discussion_r814398466)
- _findLast_ and _findLastIndex_ have been added in Chrome 97 (https://developer.chrome.com/blog/new-in-chrome-97/#new-array-prototypes)

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -1362,6 +1362,7 @@ exports.tests = [
         ie11: false,
         chrome1: false,
         chrome90: false,
+        chrome97: true,
```
Comment (ljharb) [inline data-esnext.js:1366]: might be good to confirm it's false in chrome96

**Review by ljharb — APPROVED** [review]

File: data-esnext.js (hunk 1 above)
Comment (nicolo-ribaudo) [reply data-esnext.js:1366]: I cannot find where to download Chrome 96, but it's not present in V8 9.6 (which should be the one used by Chrome 96).

Comment (ljharb) [reply data-esnext.js:1366]: verified on browserstack.

**Review by chicoxyzzy — APPROVED** [review]
Thanks Nicolò!

**Pull request merged**